messages/hu.php

```diff
@@ -14,5 +14,8 @@
 	// Model
 	'must be unique' => 'értéke már használatban van',
 	'refers to a non-existing record' => 'nem létező elemre hivatkozik',
-
+	// Html
+	'Disabled' => 'Tiltva',
+	// SamlAuth
+	'Required attribute {$attribute} is missing' => 'A kötelező "{$attribute}" attribútum hiányzik',
```

Comment:
Without quotes it would be misunderstood in some cases. You don't want to learn this language, very different...
By the way, a nice translation to French would come handy...

Reply:
I also thought this was language-specific but wanted to be sure. Would be my pleasure to provide French translations, but at some point the other colleagues might want the texts translated into their main language as well. Either way, fewer translations are a step towards many...